src/api/app/views/webui2/webui/kiwi/images/_packages.html.haml

```diff
@@ -0,0 +1,10 @@
+.card-body
+  %h3 Packages
+  #kiwi-packages-list
+    %p#no-packages{ class: "#{'hidden' if @package_groups.packages.present?}" }= 'There are no packages.'
```

Comment:
Not sure why but I have packages and this message [in the review app](https://obs-reviewlab.opensuse.org/hellcp-webui2_kiwi/kiwi_images/1/edit?section=software#). 

![screenshot from 2018-07-25 16-52-59](https://user-images.githubusercontent.com/514785/43208716-4ee6c71c-902b-11e8-9cdb-565a203bd470.png)

Reply:
Oh, forgot to replace hidden there :o